.github/workflows/validate_python.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     - name: Install poetry for validate_examples.py
       uses: abatilo/actions-poetry@v3
       with:
-        poetry-version: 1.2.2
```

Comment:
Please do not unpin Poetry. The release of Poetry 2.0 broke enough of our build pipelines as it is.

Our internal build pipelines are currently using Poetry 1.8.x. @mshafer-NI where is this documented? https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/70875/Optional-tools-for-uncommon-cases ?

I think it's fine to update Poetry to 1.8.x. Consider doing that in a separate PR.

Reply:
Yes, that's the only "doc" we have the mentions the version since preference is for devs to use the internal build tooling's "poetry" command since that allows us to bump the version by updating that. PWG is planning to evaluate and adopt 2.0 this year, but it will likely be in C2 or C3 even.